README.md

```diff
@@ -51,6 +51,10 @@ Alternatively to set it up not using Docker use below. This hosts it on `localho
 ```
 make build cypress
 ```
+## Run Cypress tests in UI
+- `make build-all`
```

Comment:
Why is this step necessary? If you are running Cypress through Yarn, it isn't using the cypress Docker container